.gitignore

```diff
@@ -144,6 +144,9 @@ goldstone/static/addons/
 addon-requirements.txt
 /addons
 /docker/**/addons/
+AUTHORS
+ChangeLog
+goldstone/compliance
```

Comment:
I'm not sure that we want to gitignore this folder.  Is this customary when dealing with submodules?  I don't believe that you should even have the folder unless you had manually installed the addon.